wiki/start/a-simple-game.md

```diff
@@ -5,605 +5,1091 @@ redirect_from:
-                <category android:name="android.intent.category.LAUNCHER" />
-            </intent-filter>
-        </activity>
-    </application>
+![Main class](/assets/images/dev/a-simple-game/4.png)
 
-</manifest>
+Declare your variables at the top of the file right underneath the `public class Main` line. You'll need a variable for every asset you plan to use:
+
+```java
+public class Main implements ApplicationListener {
+    Texture backgroundTexture;
+    Texture bucketTexture;
+    Texture dropTexture;
+    Sound dropSound;
+    Music music;
+```
+
+Enter the following code in the create method:
+
+```java
+@Override
+public void create() {
+    backgroundTexture = new Texture("background.png");
+    bucketTexture = new Texture("bucket.png");
+    dropTexture = new Texture("drop.png");
+}
 ```
 
-The setup tool already filled in the correct values for us, `android:screenOrientation` is set to "landscape". If we wanted to run the game in portrait mode we would have set that attribute to "portrait".
+This loads the assets into memory. See that the background image is loaded as a texture. Textures are the way that games keep images in video ram. It's actually not efficient to have different textures for each element in the game. It should be one big Texture for them all. Learn about [TexturePacker](/wiki/tools/texture-packer) in the wiki. For now, we'll keep these as separate textures because it's easier to explain this way.
 
-We also want to conserve battery and disable the accelerometer and compass. We do this in the `AndroidLauncher.java` file in `android/src/…` (or `drop-android`), which should look something like this:
+Similarly, we have Sound to handle the raindrop audio file in our project. Sounds are loaded completely into memory so they can be played quickly and repeatedly. Music, on the other hand, is too large to keep entirely in memory. It's streamed from the file in chunks. There is no precise rule about what should or should not be a sound versus a music, but you may consider any audio shorter than 10 seconds to be a sound.
 
 ```java
-package com.badlogic.drop;
+@Override
+public void create() {
+    ...
+    
+    dropSound = Gdx.audio.newSound(Gdx.files.internal("drop.mp3"));
+    music = Gdx.audio.newMusic(Gdx.files.internal("music.mp3"));
+}
+```
 
-import android.os.Bundle;
+We have many assets to manage now. We should use an [AssetManager](/wiki/managing-your-assets) to handle them. That can be found in the wiki too. Again, that's outside the scope of this tutorial. Let's keep it simple.
 
-import com.badlogic.gdx.backends.android.AndroidApplication;
-import com.badlogic.gdx.backends.android.AndroidApplicationConfiguration;
-import com.badlogic.drop.Drop;
+## The Game Life Cycle
+The work we've done so far is in the create method. Create executes immediately when the game is run, so of course it makes sense to load our assets there. What are these other methods for?
```

Comment:
It's a good idea to point out a common pitfall, where users create `new Texture("somePath.png")` in the constructor, at the init level, or initialized statically. This generally fails, so they should create new things in create() ! Or maybe render() occasionally!

Reply:
It's easy for me to take these kinds of details for granted now that I've been doing libGDX for awhile.